### ISSUE TITLE
Broken "left side pane" (ja-jp)

### ISSUE THREAD

**Issue by hyoshioka0128** [MicrosoftDocs/windows-itpro-docs#9432]
ja-jp:"https://docs.microsoft.com/ja-jp/windows/deployment/windows-10-poc"
![windows-10-poc_j](https://user-images.githubusercontent.com/40815708/115137968-2e5db900-a064-11eb-8edd-fb5c0f6c37e8.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b5fb2354-bc61-a5f7-3e73-456588847923
* Version Independent ID: 1825f737-8689-29b7-1ea2-b7a4d0c337fa
* Content: [Configure a test lab to deploy Windows 10 - Windows Deployment](https://docs.microsoft.com/en-us/windows/deployment/windows-10-poc)
* Content Source: [windows/deployment/windows-10-poc.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/deployment/windows-10-poc.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @greg-lindsay
* Microsoft Alias: **greglin**

**Comment by illfated** [comment]
Confirmed. This unfortunate behaviour is identical in the following pages:
- https://docs.microsoft.com/de-de/windows/deployment/windows-10-poc (Deutch)
- https://docs.microsoft.com/es-es/windows/deployment/windows-10-poc (Español)
- https://docs.microsoft.com/fr-fr/windows/deployment/windows-10-poc (Français)
- https://docs.microsoft.com/zh-cn/windows/deployment/windows-10-poc (Chinese (Simplified))
- https://docs.microsoft.com/zh-tw/windows/deployment/windows-10-poc (Chinese (Traditional))

**Comment by denisebmsft** [comment]
@TinaMcN can you please take a look?

**Comment by TinaMcN** [comment]
@denisebmsft this issue is actually caused by invalid <strong> tags in the English content, probably left over from a previous migration from .html to .md format.  <strong> is not valid for .md.  I see 153 instances of <strong> in the English content.  Can you please fix them?
![image](https://user-images.githubusercontent.com/53281468/115295388-8e109e80-a10e-11eb-8d2a-1acbcb246fda.png)

**Comment by illfated** [comment]
Raw source view for this document: https://raw.githubusercontent.com/MicrosoftDocs/windows-itpro-docs/public/windows/deployment/windows-10-poc.md

Sample from the section pictured in the comment above:
```
## Hardware and software requirements

One computer that meets the hardware and software specifications below is required to complete the guide; A second computer is recommended to validate the upgrade process.

- **Computer 1**: the computer you will use to run Hyper-V and host virtual machines. This computer should have 16 GB or more of installed RAM and a multi-core processor.
- **Computer 2**: a client computer from your corporate network. It is shadow-copied to create a VM that can be added to the PoC environment, enabling you to test a mirror image of a computer on your network. If you do not have a computer to use for this simulation, you can download an evaluation VHD and use it to represent this computer. Subsequent guides use this computer to simulate Windows 10 replace and refresh scenarios, so the VM is required even if you cannot create this VM using computer 2.

Hardware requirements are displayed below:

<div style='font-size:9.0pt'>

<table border="1" cellspacing="0" cellpadding="0">
    <tr>
        <td></td>
        <td BGCOLOR="#a0e4fa"><strong><font color="#000000">Computer 1</strong> (required)</font></td>
        <td BGCOLOR="#a0e4fa"><strong><font color="#000000">Computer 2</strong> (recommended)</font></td>
    </tr>
    <tr>
        <td BGCOLOR="#a0e4fa"><font color="#000000"><strong>Role</strong></font></td>
        <td>Hyper-V host</td>
        <td>Client computer</td>
    </tr>
    <tr>
        <td BGCOLOR="#a0e4fa"><font color="#000000"><strong>Description</strong></font></td>
        <td>This computer will run Hyper-V, the Hyper-V management tools, and the Hyper-V Windows PowerShell module.</td>
        <td>This computer is a Windows 7 or Windows 8/8.1 client on your corporate network that will be converted to a VM to demonstrate the upgrade process.</td>
    </tr>
    <tr>
        <td BGCOLOR="#a0e4fa"><font color="#000000"><strong>OS</strong></font></td>
        <td>Windows 8.1/10 or Windows Server 2012/2012 R2/2016<b>*</b></td>
        <td>Windows 7 or a later</td>
    </tr>
    <tr>
        <td BGCOLOR="#a0e4fa"><font color="#000000"><strong>Edition</strong></font></td>
        <td>Enterprise, Professional, or Education</td>
        <td>Any</td>
    </tr>
    <tr>
        <td BGCOLOR="#a0e4fa"><font color="#000000"><strong>Architecture</strong></font></td>
        <td>64-bit</td>
        <td>Any<BR><I>Note: Retaining applications and settings requires that architecture (32 or 64-bit) is the same before and after the upgrade.</I></td>
    </tr>
    <tr>
        <td BGCOLOR="#a0e4fa"><strong>RAM</strong></td>
        <td>8 GB RAM (16 GB recommended) to test Windows 10 deployment with MDT.
        <BR>16 GB RAM to test Windows 10 deployment with Microsoft Endpoint Configuration Manager.</td>
        <td>Any</td>
    </tr>
    <tr>
        <td BGCOLOR="#a0e4fa"><strong>Disk</strong></td>
        <td>200 GB available hard disk space, any format.</td>
        <td>Any size, MBR formatted.</td>
    </tr>
    <tr>
        <td BGCOLOR="#a0e4fa"><font color="#000000"><strong>CPU</strong></font></td>
        <td>SLAT-Capable CPU</td>
        <td>Any</td>
    </tr>
    <tr>
        <td BGCOLOR="#a0e4fa"><font color="#000000"><strong>Network</strong></font></td>
        <td>Internet connection</td>
        <td>Any</td>
    </tr>
</table>


<B>\*</B><I>The Hyper-V server role can also be installed on a computer running Windows Server 2008 R2. However, the Windows PowerShell module for Hyper-V is not available on Windows Server 2008 R2, therefore you cannot use many of the steps provided in this guide to configure Hyper-V. To manage Hyper-V on Windows Server 2008 R2, you can use Hyper-V WMI, or you can use the Hyper-V Manager console. Providing all steps in this guide as Hyper-V WMI or as 2008 R2 Hyper-V Manager procedures is beyond the scope of the guide.</I>
<BR>
<BR>The Hyper-V role cannot be installed on Windows 7 or earlier versions of Windows.

</div>
```
As you can see at the end of the code sample, the **bold** formatting tag `<strong> </strong>` can be replaced by `<B> </B>`, unless you want to have the entire HTML table encoding replaced with MarkDown formatting.

**Comment by TinaMcN** [comment]
@hyoshioka0128 This issue is fixed now and I think it can be closed if it looks fine for you.

**Comment by hyoshioka0128** [comment]
@TinaMcN - Fixed. Thanks.
Closed #9432 .
